syntax = "proto3";

service gRPCCommManager {
  rpc sendMessage (stream CommRequest) returns (CommResponse);
}

message CommRequest {
  int32 server_id = 1;
  string message = 2;
  string fragment_id = 3;
}

message CommResponse {
  int32 client_id = 1;
  string message = 2;
}
